.github/workflows/ci.yml

```diff
@@ -26,8 +26,9 @@ jobs:
 
       - name: Rubocop
         run: |
+          gem install rubocop rubocop-packaging --no-document
```

Comment:
`rubocop` is being installed twice (here and on the next line

Reply:
Yikes, I'm terribly sorry for this, I'll fix this.